Makefile

```diff
@@ -449,6 +449,7 @@ generate-e2e-templates: $(KUSTOMIZE) ## Generate Azure infrastructure templates
 	$(KUSTOMIZE) build $(AZURE_TEMPLATES)/v1beta1/cluster-template-node-drain --load-restrictor LoadRestrictionsNone > $(AZURE_TEMPLATES)/v1beta1/cluster-template-node-drain.yaml
 	$(KUSTOMIZE) build $(AZURE_TEMPLATES)/v1beta1/cluster-template-upgrades --load-restrictor LoadRestrictionsNone > $(AZURE_TEMPLATES)/v1beta1/cluster-template-upgrades.yaml
 	$(KUSTOMIZE) build $(AZURE_TEMPLATES)/v1beta1/cluster-template-kcp-scale-in --load-restrictor LoadRestrictionsNone > $(AZURE_TEMPLATES)/v1beta1/cluster-template-kcp-scale-in.yaml
+	$(KUSTOMIZE) build $(AZURE_TEMPLATES)/v1beta1/cluster-template-flatcar --load-restrictor LoadRestrictionsNone > $(AZURE_TEMPLATES)/v1beta1/cluster-template-flatcar.yaml
```

Comment:
instead of doing this, can we add a "test" flavor? See https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/templates/test/ci/prow-ipv6/kustomization.yaml for an example of how to do this.

The templates in infrastructure-azure/v1beta1 are only for the CAPI test suite. The Azure specific flavors for prow jobs are kept in templates/test/ci. They are based on the regular flavors with an added CRS for CNI and some resource tags to make sure the resources are cleaned up in the test sub if the test is interrupted.

Reply:
Done, I think, though I'm not sure how I can test this locally, so I just reverted this change and pushed a new one for now. Hopefully CI fails successfully again :smile:

Reply:
So the test seems to be skipped right now. Is that desired?